sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/Distribute.java

```diff
@@ -464,6 +464,13 @@ protected int getTriggerCount() {
      */
     private static final String DISTRIBUTE_GROUP = "group_distribute";
 
+    @Override
+    public void onApplicationEnterForeground() {
+        if (mChannel != null) {
+            tryResetWorkflow();
```

Comment:
1. Add log.
1. Move this method to other lifecycle callbacks (after `onActivityPaused` or so)
1. Are we sure that resume event always will be after this one?